armi/utils/__init__.py

```diff
@@ -837,6 +837,47 @@ def safeCopy(src: str, dst: str) -> None:
     runLog.extra("Copied {} -> {}".format(src, dst))
 
 
+def safeMove(src: str, dst: str) -> None:
+    """Check that a file has been successfully moved before continuing."""
+    # Convert files to OS-independence
+    src = os.path.abspath(src)
+    dst = os.path.abspath(dst)
+    if os.path.isdir(dst):
+        dst = os.path.join(dst, os.path.basename(src))
+    srcSize = os.path.getsize(src)
+    if "win" in sys.platform:
+        shutil.move(src, dst)
+    elif "linux" in sys.platform:
+        cmd = f'mv "{src}" "{dst}"'
+        os.system(cmd)
+    else:
+        raise OSError(
+            "Cannot perform ``safeMove`` on files because ARMI only supports "
+            + "Linux and Windows."
+        )
```

Comment:
We run unit tests on mac OS now. So is this OSError (here and in other places) accurate? @john-science

Reply:
(keep in mind Mike is just following the "template" created in `safeCopy`)

Reply:
@mgjarrett ARMI CI runs on Mac, Linux, and Windows. Yet I don't see this error being raised when the unit tests run on MacOS.  (Your CI passes in this PR.)

What's going on?

Reply:
Also, I note, every single other feature in ARMI supports the MacOS. I'm not sure why this would be different.

Reply:
@john-science before macos was added, we added this exact if/elif/else configuration to several places in ARMI. You'll note it is in `safeCopy` just above this in this file.

I'm not sure why the test passes on macos, but note that `test_safeCopy` also passes and it's the same structure.